### ISSUE TITLE
Does the tutorial need to be updated now we have ES8388 driver support?

### ISSUE THREAD

**Issue by mrSilkie** [grame-cncm/faust#489]
https://faust.grame.fr/doc/tutorials/#configuring-the-audio-codec

states no driver support for es8388. [A new pull](https://github.com/grame-cncm/faust/pull/483) suggests that this is no longer the case. 

Can somebody give an example of how they would configure the new drivers?

**Comment by droosenb** [comment]
Hi mrSilkie!

Yes, it is now the case that faust supports the es8388. The drivers are currently tested for the espressif LyraT 4.2, and intended for use specifically on that board. If you know of another board using the es8388, some specific parameters may need to be changed based on how the codec is physically connected to the esp32. 

Sorry about the tutorial, its quite outdated and is missing several new updates. I'll talk to Romain about getting permission to update it with new information. 

I've uploaded a simple configuration example I've tested on the LyraT using the new drivers. It should (hopefully) contain all the relevant information to create new projects. 

https://github.com/droosenb/faust-on-ESP32

es8388-configuration-example is the project you're looking for. Let me know if you have any issues.
